doc/policy/packages.md

```diff
@@ -117,3 +118,30 @@ backward-compatible for users and applications that rely on p2p transaction rela
 package validation should not prevent the acceptance of a transaction that would otherwise be
 policy-valid on its own. By always accepting a transaction that passes individual validation before
 trying package validation, we prevent any unintentional restriction of policy.
+
+### Package Replace by Fee
+
+Child-with-unconfirmed-parents packages may replace mempool transactions under the following conditions:
+
+1. The child transaction does not conflict with any transactions in the mempool.
+
+2. The directly conflicting transactions all signal replaceability explicitly.
+
+3. All transactions in the package, including those that do not directly conflict with any mempool
+   transactions, only include an unconfirmed input if that input was included in one of the directly
+   conflicting transactions or is from another transaction in the package.
+
+4. The package fee after deduplication pays an absolute fee of at least the sum paid by the
+   original transactions.
+
+5. The additional fees (difference between absolute fee paid by the package after deduplication and the
```

Comment:
Depending on the design of p2p packages, we might relay the "full-non-dedup" version of the package as our peers might not have the same mempool composition. So the additional fees might have to be paid on the non-dedup package. I don't think we have to settle the question now though we might have to rework package-RBF in function of p2p design ?